specification/security_model.adoc

```diff
@@ -1,9 +1,195 @@
 [[security_model]]
-== Security and Threat Model
+== Security Model
+
+The CoVE-IO securiy model is built on the following assumptions:
+
+- Only the TVM owner can assess of a TDI trustworthiness. Based on that
+  assessment, it explicitly accepts or rejects a TDI into its TCB.
+- A TDI may access a TVM confidential memory only when the following conditions
+  are met:
+  1. The TVM owner has explicitly allowed the TDI to access its memory by
+     accepting it.
+  2. The TDI is exclusively assigned to the TVM, i.e. it must not be shared
+     other TVMs or any host software component.
+- A TVM may trust and access a TDI MMIO space if and only if the TDI is
+  exclusively assigned to the TVM.
```

Comment:
I feel this sentence need reword.

The original sentence implies that "as long as TDI is exclusively assigned, then the TVM may trust it." I don't think that is true, because the TVM need accept the TDI as well as prerequisite.

I think DMA and MMIO should have similar rule:
1. The TVM owner has explicitly accepted the TDI.
2. The TDI is exclusively assigned to the TVM.